home/WEB-INF/jsp/publication/publication-view.jsp

```diff
@@ -84,126 +43,123 @@
     </jsp:attribute>
 
     <jsp:body>
-
-        <div id="${zfn:makeDomIdentifier(SUMMARY)}">
+        <div id="${zfn:makeDomIdentifier(NavigationMenuOptions.SUMMARY.value)}">
             <div class="small text-uppercase text-muted">PUBLICATION</div>
             <h1>${publication.title}</h1>
             <jsp:include page="publication-view-summary.jsp"/>
         </div>
 
-        <z:section title="${ABSTRACT}">
+        <z:section title="${NavigationMenuOptions.ABSTRACT.value}">
             <zfin2:subsection title="" test="${not empty abstractText}" showNoData="true">
                 ${abstractText}
             </zfin2:subsection>
         </z:section>
 
-        <z:section title="${GENES}">
+        <z:section title="${NavigationMenuOptions.ERRATA.value}"
```

Comment:
maybe instead of passing title and hide, just pass the navmenu and the ERRATA and let the section tag handle the logic?